locations/spiders/rivian_us.py

```
@@ -0,0 +1,210 @@
+
+class RivianUSSpider(Spider):
+    """
+    Rivian Adventure Network DC fast chargers and Rivian Waypoints destination
+    chargers, from the vector tiles behind https://rivian.com/experience/charging
+    """
+
+    name = "rivian_us"
+    custom_settings = {"DOWNLOAD_DELAY": 0}
+    item_attributes = {"operator": "Rivian", "operator_wikidata": "Q7338847"}
+    allowed_domains = ["api.rivianservices.com"]
+    seen_refs: set[str] = set()
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)